packages/parity-electron/README.md

```diff
@@ -76,3 +76,7 @@ If Parity has been downloaded to Electron's `userData` folder, then it deletes t
 #### `signerNewToken(): Promise<String>`
 
 Runs `parity signer new-token` and resolves with a new secure token to be used in a dapp. Rejects if no token could be extracted.
+
+#### `checkClockSync(): Promise<Object{isClockSync: boolean, timeDrift: number}>`
```

Comment:
The first variable is called `isClockSync` here, `isSync` in the code, and `clockSync` in the healthStore. They should probably all have the same name.